script.js

```diff
@@ -16,26 +23,55 @@ restaurantsApp.getRestaurantIDs = (searchTerm, searchLocation) => {
         },
         data: {
             term: searchTerm,
-            location: searchLocation
+            location: searchLocation,
+            limit: 3,
         }
     }).then(function (result) {
-        //This takes the first three results, can be changed later
-        for (let i = 0; i < 3; i++) {
-            let businessID = result.businesses[i].id
-            let businessName = result.businesses[i].name
-            let businessRating = result.businesses[i].rating
-            let businessPrice = result.businesses[i].price
-            let businessAddress = result.businesses[i].location.display_address
-        }
-    });
+        console.log(result)
+        // result.bussinesses is an array on bussinesses that can be looped through with forEach method ,then for each business object the info are stored in respective array
+        result.businesses.forEach(function(item){
+
+             restaurantsApp.businessID.push(item.id)
+             restaurantsApp.businessRating.push(item.rating)
+             restaurantsApp.businessPrice.push(item.price)
+             restaurantsApp.businessName.push(item.name)
+             restaurantsApp.businessAddress.push(item.location.display_address) 
+          
+    }); 
+    restaurantsApp.getReviews()
```

Comment:
based on our discussion we should invoke this as:

```
restaurantsApp.getReviews(businessIDs)
```

Reply:
got it